Utils/Dataflow/README

```diff
@@ -133,8 +133,35 @@ following requirements:
 
 [3] EOP, End-of-Process marker. Default value: '\0' (NULL)
 
+==================
+* 3. Stage types *
+==================
+There are three types of stages:
+
+1. Source stage - an initial stage in a dataflow pipeline that gets data
+from an external source (CDS, Rucio, AMI, etc.).
+
+2. Transform stage - a transitional stage that performs single logical
+operation on the metadata. Set (chain) of transform stages forms the whole
+transformation process for messages emitted by Source. Transformation
```

Comment:
"Emitted" is usually applied to energy/light/etc. How about "produced", "constructed", or something else?

Reply:
@Evildoor 
I think, "produced" may be correct enough for the context